src/actions/TemperatureSyncActions.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2020
+ */
+
+import { generateUUID } from 'react-native-database';
+
+import { UIDatabase } from '../database';
+import { Sensor } from '../database/DataTypes';
+
+export const TEMPERATURE_SYNC_ACTIONS = {
+  SCAN_START: 'TemperatureSync/scanStart',
+  SCAN_COMPLETE: 'TemperatureSync/scanComplete',
+  SCAN_ERROR: 'TemperatureSync/scanError',
+};
+
+const scanStart = () => ({ type: TEMPERATURE_SYNC_ACTIONS.SCAN_START });
+const scanComplete = () => ({ type: TEMPERATURE_SYNC_ACTIONS.SCAN_COMPLETE });
+const scanError = () => ({ type: TEMPERATURE_SYNC_ACTIONS.SCAN_ERROR });
+
+const updateSensors = sensorAdvertisements => dispatch => {
```

Comment:
this is working well. A good advantage to this is we don't need to throw up the spinner while it's scanning. In the page where we display and hook up sensors to locations, we can hook into the `scanComplete()` action to refresh that page with the new sensors